RELEASING.md

```diff
@@ -0,0 +1,13 @@
+# Releasing fullstory-browser-sdk on npm
+
```

Comment:
Members of the [JS SDK Contributors team](https://github.com/orgs/fullstorydev/teams/js-sdk-contributors) can deploy to NPM.